src/components/lib/cos_kernel_api.c

```diff
@@ -649,6 +649,10 @@ int
 cos_thd_switch(thdcap_t c)
 { return call_cap_op(c, 0, 0, 0, 0, 0); }
 
+int
+cos_thd_wakeup(thdcap_t thd, tcap_t tc, tcap_prio_t prio)
+{ return call_cap_op(tc, CAPTBL_OP_TCAP_WAKEUP, thd, (prio << 32) >> 32, prio >> 32, 0); }
```

Comment:
BTW, I doubt the shift vs. mask thing is going to generate any different code (assuming unsigned values).  The question is what is easier to read.  If we macro-ify this, then it doesn't really matter, so I don't have a strong opinion either way.

I had figured that the shifts were easier to read before, but with a macro, I don't think it matters.